scripts/create-tables.sql

```diff
@@ -56,6 +55,13 @@ CREATE TABLE IF NOT EXISTS "elements"
   CONSTRAINT elements_id_pk PRIMARY KEY (id)
 );
 
+/* Unique indexes */
+CREATE UNIQUE INDEX pages_project_id_x_y_key ON pages
+(project_id, x, y) WHERE deleted_at IS NULL;
+
+CREATE UNIQUE INDEX pages_project_id_x_y_deleted_at_key ON pages
+(project_id, x, y, deleted_at) WHERE deleted_at IS NOT NULL;
```

Comment:
Why would they fail? Wouldn't the deleted_at value be different for each deleted versions?